### DISCUSSION TITLE
Paypal Pay Later support for Cart and Checkout Blocks.

### DISCUSSION THREAD

**Discussion by houseandhaven** [woocommerce/woocommerce#45089] [WooCommerce, Blocks, and Site Editing]
Hi there,

I would like to know if there is anyway you can add the Paypal pay later option to this?

I know the Paypal standard works fine which is great but I'd like to use the pay later option that is on the WooCommerce PayPal Payments plugin.

Can you help at all with this?

Thanks

**Comment by nerrad** [comment]
Hi there @houseandhaven 👋🏻 

Thanks for leaving your feedback here. WooCommerce PayPal Payments is currently not supporting Cart and Checkout blocks but it is something known by the team that works on PayPal Payments. There is no ETA on when support will be added.

I'm going to close this issue because there's currently nothing actionable by the team working on WooCommerce Blocks. Thanks again for letting us know about your interest in having this supported!

**Comment by lecombat** [comment]
Hi all,

I have same issue with Paypal Paylater, the payment method not display in a woocommerce checkout page (even configuration are good). 

Did you have some alternative for this issue please ?

Best regards,
